### DISCUSSION TITLE
What is the standard end device OTA file extension

### DISCUSSION THREAD

**Discussion by hassan-xy** [zwave-js/zwave-js-ui#3717] [Q&A] [answered]
Hi team,
I have noticed 1 device type OTA extension as .gbl while for another 1 as .otz . I am not sure how many more are there. Please share any documentation or update as to how many extensions the end-device ota supports.

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by AlCalzone** [comment, marked as answer]
https://zwave-js.github.io/node-zwave-js/#/api/node?id=updatefirmware

- "aeotec" - A Windows executable (.exe or .ex_) that contains Aeotec's upload tool
- "otz" - A compressed firmware file in Intel HEX format
- "ota" or "hex" - An uncompressed firmware file in Intel HEX format
- "hec" - An encrypted Intel HEX firmware file
- "gecko" - A binary gecko bootloader firmware file with .gbl extension

**Reply by AlCalzone** [reply]
.gbl are for 700+ series devices, older devices typically use one of the other extensions.

**Comment by hassan-xy** [comment]
Thanks for the quick response.
